Source/DafnyCore/AST/Grammar/TokenNewIndentCollector.cs

```diff
@@ -692,6 +701,10 @@ private class Indentations {
       }
     }
 
+    if (stmt is AssignSuchThatStmt assignSuchThatStmt) {
+      Visit(assignSuchThatStmt.Expr, rightIndent);
```

Comment:
We were missing the visit of the right-hand-side of an assign such that statement. Now it's being tested